src/storage/v2/storage.cpp

```diff
@@ -418,14 +411,29 @@ Storage::Storage(Config config)
     commit_log_.emplace(timestamp_);
   }
 
-  if (config_.durability.restore_replicas_on_startup) {
-    spdlog::info("Replica's configuration will be stored and will be automatically restored in case of a crash.");
+  if (config_.durability.restore_replication_state_on_startup) {
+    spdlog::info("Replication configuration will be stored and will be automatically restored in case of a crash.");
     utils::EnsureDirOrDie(config_.durability.storage_directory / durability::kReplicationDirectory);
     storage_ =
         std::make_unique<kvstore::KVStore>(config_.durability.storage_directory / durability::kReplicationDirectory);
-    RestoreReplicas();
+
+    RestoreReplicationRole();
```

Comment:
I see here we are restoring replication role on startup, but why are we restoring replica only for MAIN?

Reply:
we are restoring replication role, and restoring corresponding replicas if it's the main